Lib/subprocess.py

```diff
@@ -134,6 +134,7 @@ def __init__(self, *, dwFlags=0, hStdInput=None, hStdOutput=None,
             self.hStdOutput = hStdOutput
```

Comment:
I think the caller would expect the handles he passes to be inherited and not a duplicate of them. The usage is to be able to selectively inherit some specific handles after all. A script using this is very likely to pass the handles he passed to `handle_list` in the command line or some other IPC mechanism for the usage of the sub process, otherwise what is the point of inheriting those handles besides leaking them?

Imagine a process opening some anonymous shared memory and passing that handle on the command line for a new process, making sure to properly inherit it via `handle_list`. The new process will have access to the shared memory and no other additional handles which it won't even know about, unless any others are passed.